Make the use of SCA optional (#1622)
theforeman/foreman-documentation: guides/common/modules/proc_attaching-satellite-infrastructure-subscription.adoc

= Attaching the {Project} Infrastructure Subscription

> NOTE: Skip this section if the Simple Content Access (SCA) is already in use.

Reviewer: ```suggestion
NOTE: Skip this section if you are already using the Simple Content Access (SCA).
```
Active voice :)

Reviewer: Or even better (as [in 6.11](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/installing_capsule_server/installing-capsule-server#attaching-infrastructure-subscription_capsule))
```suggestion
NOTE: Skip this section if you enabled the Simple Content Access (SCA) in the Red{nbsp}Hat Customer Portal.
```

> NOTE: Skip this section if the Simple Content Access (SCA) is already in the use.

Reviewer 2: ```suggestion
NOTE: Skip this section if the Simple Content Access (SCA) is already in use.
```

= Attaching the {Project} Infrastructure Subscription

> NOTE: Skip this section if the Simple Content Access (SCA) is already in the use.
> This section should be only followed in the case of SCA is not in use.

Reviewer 3: ```suggestion
NOTE: Skip this section if the Simple Content Access (SCA) is already in the use.
```
The first sentence seems sufficient enough.
